Do not include statically linked libstdc++. Not a good idea if we don't need
to be binary-compatible across exotic distribution. Doesn't work with trunk
glibc heading towards 2.11 anyways. Static linking is not really supported
anyways.

Lubomir Rintel <lkundrak@v3.sk>

diff -up VirtualBox-3.1.0_OSE/src/VBox/Additions/x11/VBoxClient/Makefile.kmk.libcxx VirtualBox-3.1.0_OSE/src/VBox/Additions/x11/VBoxClient/Makefile.kmk
--- VirtualBox-3.1.0_OSE/src/VBox/Additions/x11/VBoxClient/Makefile.kmk.libcxx	2009-11-05 17:54:28.000000000 +0100
+++ VirtualBox-3.1.0_OSE/src/VBox/Additions/x11/VBoxClient/Makefile.kmk	2009-11-12 15:37:07.000000000 +0100
@@ -68,20 +68,20 @@ endif
 ifdef VBOX_WITH_GUEST_PROPS
  VBoxClient_DEFS += VBOX_WITH_GUEST_PROPS
 endif
-#
-# Link aginst libstdc++.a. (http://www.trilithium.com/johan/2005/06/static-libstdc/).
-# (It would've been preferred to avoid features depending on libstdc++, of course...)
-#
-# Actually, this is darn annoying and will *NOT* be tolerated for any new code!
-#
-VBoxClient_LIBPATH  += $(PATH_VBoxClient)
-VBoxClient_ORDERDEPS = $(PATH_VBoxClient)/libstdc++.a
-VBoxClient_CLEAN     = $(PATH_VBoxClient)/libstdc++.a
-$$(PATH_VBoxClient)/libstdc++.a:
-	$(call MSG_L1,Forcing static libstdc++)
-	$(QUIET)$(MKDIR) -p $(@D)
-	$(QUIET)$(LN_EXT) -sf `$(TOOL_$(VBOX_GCC_TOOL)_CXX) $(TEMPLATE_VBOXGUESTR3CPPEXE_CXXFLAGS.$(KBUILD_TARGET_ARCH)) -print-file-name=libstdc++.a` $@ \
-	     || $(CP_EXT) -f  `$(TOOL_$(VBOX_GCC_TOOL)_CXX) $(TEMPLATE_VBOXGUESTR3CPPEXE_CXXFLAGS.$(KBUILD_TARGET_ARCH)) -print-file-name=libstdc++.a` $@
+##
+## Link aginst libstdc++.a. (http://www.trilithium.com/johan/2005/06/static-libstdc/).
+## (It would've been preferred to avoid features depending on libstdc++, of course...)
+##
+## Actually, this is darn annoying and will *NOT* be tolerated for any new code!
+##
+#VBoxClient_LIBPATH  += $(PATH_VBoxClient)
+#VBoxClient_ORDERDEPS = $(PATH_VBoxClient)/libstdc++.a
+#VBoxClient_CLEAN     = $(PATH_VBoxClient)/libstdc++.a
+#$$(PATH_VBoxClient)/libstdc++.a:
+#	$(call MSG_L1,Forcing static libstdc++)
+#	$(QUIET)$(MKDIR) -p $(@D)
+#	$(QUIET)$(LN_EXT) -sf `$(TOOL_$(VBOX_GCC_TOOL)_CXX) $(TEMPLATE_VBOXGUESTR3CPPEXE_CXXFLAGS.$(KBUILD_TARGET_ARCH)) -print-file-name=libstdc++.a` $@ \
+#	     || $(CP_EXT) -f  `$(TOOL_$(VBOX_GCC_TOOL)_CXX) $(TEMPLATE_VBOXGUESTR3CPPEXE_CXXFLAGS.$(KBUILD_TARGET_ARCH)) -print-file-name=libstdc++.a` $@
 
 ifdef VBOX_X11_SEAMLESS_GUEST
  if defined(VBOX_WITH_TESTCASES) && !defined(VBOX_ONLY_ADDITIONS) && !defined(VBOX_ONLY_SDK)
